nextflow_schema.json

```
@@ -570,24 +570,24 @@
             },
             "fa_icon": "fas fa-layer-group"
         },
-        "stripping": {
-            "title": "Stripping",
+        "host_removal": {
+            "title": "Host Removal",
```

```suggestion
            "title": "Removal of Host-Mapped Reads",
```
To sync with title change in `usage.md`